v2/samples/containerservice/v1api20230202preview/v1api20230202preview_trustedaccessrolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: containerservice.azure.com/v1api20230202preview
+kind: TrustedAccessRoleBinding
+metadata:
+  name: sampltrustedaccessrolebinding202221preview
+  namespace: default
+spec:
+  location: westus3
+  owner:
+    name: samplemanagedcluster202221preview
+    roles: 
+      - Microsoft.MachineLearningServices/workspaces/mlworkload
```

Comment:
Is `Microsoft.MachineLearningServices/workspaces/mlworkload` a role? Shouldn't it be something like `reader`, `owner`, or `contributor`?

Reply:
Yup, that's the name of the role. You can use `az aks trustedaccess rolebinding list` to see what roles are available for a particular cluster.